src/Extensions/Logging/Internal/EventSource/BasicMachineInformation.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	internal class BasicMachineInformation : EmptyMachineInformation
+	{
+		public BasicMachineInformation()
+		{
+			MachineName = GetMachineName();
+			MachineId = MachineName;
```

Comment:
Why do we need this field?

Reply:
It's useless in this context but useful for service fabric context for example when we have multiple machines with the same name